cmd/ci-images-mirror/main.go

```diff
@@ -83,6 +87,8 @@ func newOpts() *options {
 	fs.IntVar(&opts.port, "port", 8090, "Port to run the server on")
 	fs.DurationVar(&opts.gracePeriod, "gracePeriod", time.Second*10, "Grace period for server shutdown")
 	fs.BoolVar(&opts.onlyValidManifestV2Images, "only-valid-manifest-v2-images", true, "If set, source images with invalidate manifests of v2 will not be mirrored")
+	fs.BoolVar(&opts.validateConfigOnly, "validate-config-only", false, "If set, only validate the config file and exit")
+	fs.StringVar(&opts.ciOperatorconfigPath, "ci-operator-config-path", "", "Path to the ci operator config")
```

Comment:
This is not really needed.
It is calculated the code already.
https://github.com/openshift/ci-tools/blob/1420774f51896ba02c21ce2c6dab35f39fa8db89/cmd/ci-images-mirror/main.go#L277